build.gradle

```diff
@@ -55,7 +55,7 @@ dependencies {
     final javadocJDKFiles = files(((URLClassLoader) ToolProvider.getSystemToolClassLoader()).getURLs())
     compileOnly(javadocJDKFiles)
     testCompile(javadocJDKFiles)
-    compile 'org.freemarker:freemarker:2.3.18'
+    compile 'org.freemarker:freemarker:2.3.23'
```

Comment:
@cmnbroad I just tried a quick clean build - we'll need to make some code changes to make it compile (the deprecated warnings are causing the build to fail).

I think upgrading is a good idea, we'll just need to do some extra legwork to make sure we're using the most up-to-date interfaces in to FreeMarker (to fix the warnings) and I'm not sure what the full list is.